mkdocs.yml

```diff
@@ -353,7 +358,6 @@ nav:
               - 'SATA hot plug': unified-test-documentation/dasharo-compatibility/346-SATA-hotplug-detection.md
               - 'Sign of life': unified-test-documentation/dasharo-compatibility/347-sign-of-life.md
               - 'BIOS menu function keys': unified-test-documentation/dasharo-compatibility/357-bios-menu-function-keys.md
-              - 'Suspend Mechanism Switching S0ix/S3': unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md
```

Comment:
This change is irrelevant, please revert.

Reply:
I reverted the change.